cli/driver/src/exporter.rs

```diff
@@ -390,6 +390,16 @@ impl Callbacks for ExtractionCallbacks {
                     }
                 }
                 ExporterCommand::Backend(backend) => {
+                    match backend.backend {
+                        hax_cli_options::Backend::Easycrypt
+                        | hax_cli_options::Backend::ProVerif => {
+                            eprintln!(
+                                "⚠️ Warning: Experimental backend \"{}\" is work in progress.",
+                                backend.backend
+                            )
+                        }
+                        _ => (),
+                    };
```

Comment:
Small (purely syntactical) nit: you could write that with the [`matches` macro](https://doc.rust-lang.org/std/macro.matches.html)
```rust
use hax_cli_options::Backend;
if matches!(backend.backend, Backend::Easycrypt | Backend::ProVerif) {
  ...
}

Reply:
Thanks for pointing that out! That's much more concise. :+1:

Reply:
:smiley: